ipyvolume/pylab.py

```diff
@@ -367,12 +367,12 @@ def plot(x, y, z, color=default_color, visible_lines=True, color_selected=None,
     :param y: {y}
     :param z: {z}
     :param color: {color}
-    :param visible_lines: {visible_lines}
-    :param color_selected: {color_selected}
```

Comment:
This arg would only make sense for scatter, unless we want plot and scatter to have the same arguments but different defaults, that may not be such a bad idea actually.